sockeye/inference.py

```diff
@@ -462,18 +470,21 @@ class LengthPenalty:
     def __init__(self, alpha: float = 1.0, beta: float = 0.0) -> None:
         self.alpha = alpha
         self.beta = beta
-        self.denominator = (self.beta + 1) ** self.alpha
+        self.denominator = float((self.beta + 1.) ** self.alpha)
```

Comment:
Is `float()` needed if both operands are floats?

Reply:
yeah it's probably not necessary especially as with python3.6 there's a special int division operator `//`. I'll revert this.